src/test/resources/proxy/requests-search.json

```diff
@@ -226,5 +226,92 @@
         ]
       }
     }
+  },
+  {
+    "name": "GET search from root with queryparameter",
+    "requestGateway": {
+      "method": "GET",
+      "url": "_search?q=age:>33",
```

Comment:
Dont we want the index here?
index-3/_search

Reply:
No it was intended to test without indexName ( i.e analog to *)